automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/StorageApiSpec.scala

```diff
@@ -54,43 +54,6 @@ class StorageApiSpec extends AnyFreeSpec with StorageApiSpecSupport with Matcher
   // check for a 403 when requesting non-existent objects, which may seem wrong at first glance to anyone
   // reading this code.
 
```

Comment:
This note here says that these tests are checking that errors get propagated to the user.  Would we want to have one test here that mocks google with an error code just to check error propagation?

Reply:
We could add something like 
```
  "should propagate exception" in {
      val error = intercept[FireCloudExceptionWithErrorReport] {
        Await.result(storageService.getObjectStats("exception", "bar"), Duration.Inf)
      }
      assert(error.getMessage.contains("No access"))
    }
```
But I'm not convinced it tests much worthwhile.  Were you thinking along those ^^ lines or something else?

Reply:
Sure, I was just trying to honor the comment.  If it's not worth it, I'm not attached to it.